client/src/pages/CreateSurvey/Questions/Question.jsx

```diff
@@ -23,11 +23,42 @@ const Question = ({ questionIndex }) => {
     (state) => state.createSurveyReducer.questions[questionIndex],
   );
   const { questions } = useSelector((state) => state.createSurveyReducer);
-
+  const errors = useSelector((state) => state.errorsBagReducer);
   const dispatch = useDispatch();
 
-  const setQuestionData = (event, inputType) => {
+  const setQuestionData = (
+    event,
+    inputType,
+    name,
+    value,
+    min,
+    max,
+    questionIdentifier,
+  ) => {
     event.persist();
+
+    if (inputType === 'text') {
+      const questionTitleIsBiggerThanMaxValue =
+        event.target.value && event.target.value.length > max;
+      const questionTitleIsSmallerThanMinValue =
+        event.target.value && event.target.value.length < min;
+      const questionTitleIsEmpty = event.target.value === '';
+      if (
+        questionTitleIsBiggerThanMaxValue ||
+        questionTitleIsSmallerThanMinValue ||
+        questionTitleIsEmpty
```

Comment:
You use this conditional quite a few times throughout this file and it spans multiple lines. Maybe you should write it as one single conditional that can be re-used everywhere.